src/windows/Git-Credential-Manager.UI.Windows/Git-Credential-Manager.UI.Windows.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net472</TargetFramework>
+    <UseWPF>true</UseWPF>
+    <RootNamespace>GitCredentialManager.UI</RootNamespace>
+    <AssemblyName>git-credential-manager-ui</AssemblyName>
+    <ApplicationIcon>$(RepoAssetsPath)gcmicon.ico</ApplicationIcon>
+    <StartupObject>GitCredentialManager.UI.Program</StartupObject>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\shared\Core.UI\Core.UI.csproj" />
+    <ProjectReference Include="..\Core.UI.Windows\Core.UI.Windows.csproj" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(OSPlatform)' != 'windows'">
```

Comment:
Will this ever be the case?

Reply:
Yes, sadly.. opening the solution in _some_ IDEs (\*cough\* VSMac \*cough\*) the projects are still built/restored even if the solution configuration isn't in WindowDebug/Release... :(